.travis.yml

```diff
@@ -158,6 +158,8 @@ script:
 - tox
 
 after_failure:
+- ip a
+- sysctl -a
```

Comment:
@jaraco

> You say that one can emulate this on Linux by disabling IPv6 - is that what's happening in Travis?

Yes, and I've added this output to confirm that IPv6 is disabled.

Reply:
@jaraco please note that the same issue happens in CircleCI: https://circleci.com/gh/cherrypy/cheroot/223